src/framework/theme/components/spinner/_spinner.component.theme.scss

```diff
@@ -0,0 +1,139 @@
+@mixin nb-spinner-theme() {
+  nb-spinner {
+    opacity: 1;
+    position: absolute;
+    /*
+    background-color: nb-theme(card-header-disabled-bg);
+    */
+    border-radius: inherit;
+    width: 100%;
+    height: 100%;
+    z-index: 6000;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    transition: 0.3s ease-in-out;
+
+    &.active-spinner .spin-circle {
+      border-right-color: nb-theme(color-active);
```

Comment:
we need to have property-mediators for each component, so `spinner-fg-active`, `spinner-fg-primary`, etc, same as for cards for example